server/src/test/java/io/druid/server/lookup/cache/LookupCoordinatorManagerTest.java

```diff
@@ -1245,6 +1254,8 @@ public void testStartTooMuch() throws Exception
         lookupCoordinatorManagerConfig
     );
     manager.start();
+    Assert.assertTrue(manager.backgroundManagerIsRunning());
+    Assert.assertFalse(manager.waitForBackgroundTermination(10));
```

Comment:
minor nit: could we also test post stop semantics, i.e backgroundManagerIsRunning is false & waitForBackgroundTermination returning true